test/dsl/dsl_model_construction.jl

```diff
@@ -122,17 +123,16 @@ let
     push!(identical_networks_1, reaction_networks_standard[8] => different_arrow_8)
 
     for networks in identical_networks_1
-        f1 = ODEFunction(convert(ODESystem, networks[1]), jac = true)
-        f2 = ODEFunction(convert(ODESystem, networks[2]), jac = true)
-        g1 = SDEFunction(convert(SDESystem, networks[1]))
-        g2 = SDEFunction(convert(SDESystem, networks[2]))
         for factor in [1e-2, 1e-1, 1e0, 1e1, 1e2, 1e3]
-            u0 = factor * rand(rng, length(get_unknowns(networks[1])))
-            p = factor * rand(rng, length(get_ps(networks[1])))
+            u0_1 = rnd_u0(networks[1], rng; factor)
```

Comment:
Uses a helper function that creates a random u0 (another one does ps) vector (and as a symbolic map).